packages/react-test-app/src/preview-examples/modal-form-ix-button-submit.tsx

```diff
@@ -0,0 +1,69 @@
+import {
+  IxButton,
+  IxModalContent,
+  IxModalFooter,
+  IxModalHeader,
+  Modal,
+  ModalRef,
+  showModal,
+} from '@siemens/ix-react';
+import { useRef } from 'react';
+
+function CustomModal() {
+  const modalRef = useRef<ModalRef>(null);
+
+  const close = () => {
+    modalRef.current?.close('close payload!');
+  };
+  const dismiss = () => {
+    modalRef.current?.dismiss('dismiss payload');
+  };
+
+  return (
+    <Modal ref={modalRef}>
+      <IxModalHeader onCloseClick={() => dismiss()}>
+      Create Resource
+      </IxModalHeader>
+      <IxModalContent>
+         
+          
+          <form id="create-resource-form" onSubmit={(e) => {
+            e.preventDefault();
+            console.log('Form submitted successfully');
+            close();
+          }}>
+            <div className="form-group">
+              <label htmlFor="name">Name</label>
+              <input
+                type="text"
+                className="form-control"
+                id="name"
+                name="name"
+                
+              />
+            </div>
+            </form>
+          
+        
+      </IxModalContent>
```

Comment:
- Please use tools like prettier to format your code.
- Use `ix-input` inside example not the standard elements (label + input)